src/Raven.Server/Documents/Indexes/Static/StaticIndexBase.cs

```diff
@@ -560,7 +560,10 @@ private static object VectorFromText(IndexField indexField, object value)
             {
                 var vectorList = new List<VectorValue>();
                 foreach (var item in enumerable)
+                {
```

Comment:
What if the `item` is `null`? It would throw an exception now. Should we return a list like `[vec0, vec2, VectorValue.Null, [...], vecN]` instead?